styles.css

```diff
@@ -0,0 +1,60 @@
+body{
+max-width:100%;
+max-height:100%;
+background-color: rgb(92, 12, 38);
+
+
+}
+main{
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+.headline{
+    width: 50%;
+    height: 50px;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    font-size: 50px;
```

Comment:
You haven't been following needed style guideline
 http://codeguide.academy/html-css.html